ush/fill_jinja_template.py

```diff
@@ -270,6 +271,9 @@ def fill_jinja_template(argv):
     # Read in the config options from the provided (optional) YAML file
     cfg = cla.config if cla.config is not None else {}
 
+    if config_dict is not None:
+        update_dict(cfg, config_dict)
```

Comment:
Need to allow output to be suppressed when quiet option is active

```suggestion
        update_dict(cfg, config_dict, quiet=cla.quiet)
```